docs/index.rst

```diff
@@ -59,5 +59,6 @@ Documentation Contents
     api/index.rst
     get-help
     contributing
+    maintaining
```

Comment:
WDYT about linking to this from the Contributing page instead of having it on the sidebar?